### PR TITLE
sriov: refactored list_test.go

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

**Review by sebrandon1 — APPROVED** [review]
/lgtm

File: pkg/sriov/list_test.go [hunk 1]
```diff
@@ -206,47 +208,47 @@ func TestListNetworkNodeState(t *testing.T) {
...
+				buildDummySrIovPolicy("test1", "testnamespace")},
+			nsName: "testnamespace",
 			client: true,
 		},
 		{
 			testNetworkNodeStates: []*srIovV1.SriovNetworkNodePolicy{
-				buildDummySrIovPolicy("test", defaultNodeNsName),
-				buildDummySrIovPolicy("test1", defaultNodeNsName)},
+				buildDummySrIovPolicy("test", "testnamespace"),
+				buildDummySrIovPolicy("test1", "testnamespace")},
```
Comment (trewest) [inline pkg/sriov/list_test.go:226]: Can you add nsName: "" ?

File: pkg/sriov/list_test.go [hunk 2]
```diff
@@ -278,47 +280,47 @@ func TestListPolicy(t *testing.T) {
...
+			nsName: "testnamespace",
 			client: true,
 		},
 		{
 			testNetworkPolicy: []*srIovV1.SriovNetworkNodePolicy{
-				buildDummySrIovPolicy("test", defaultNodeNsName),
-				buildDummySrIovPolicy("test1", defaultNodeNsName)},
+				buildDummySrIovPolicy("test", "testnamespace"),
+				buildDummySrIovPolicy("test1", "testnamespace")},
 			expectedError: fmt.Errorf("failed to clean up SriovNetworkNodePolicies, 'operatornsname' parameter is empty"),
```
Comment (trewest) [inline pkg/sriov/list_test.go:299]: Can you add nsName: "" ?

File: pkg/sriov/list_test.go [hunk 3]
```diff
@@ -139,42 +141,42 @@ func TestNetworkCleanAllNetworksByTargetNamespace(t *testing.T) {
...
+				buildNodeNetworkState("test2", "testnamespace")},
+			nsName: "testnamespace",
 			client: true,
 		},
 		{
-			testNetworkNodeStates: []*srIovV1.SriovNetworkNodeState{buildNodeNetworkState("test", defaultNodeNsName),
-				buildNodeNetworkState("test2", defaultNodeNsName)},
+			testNetworkNodeStates: []*srIovV1.SriovNetworkNodeState{buildNodeNetworkState("test", "testnamespace"),
+				buildNodeNetworkState("test2", "testnamespace")},
 			expectedError: fmt.Errorf("failed to list SriovNetworkNodeStates, 'nsname' parameter is empty"),
```
Comment (trewest) [inline pkg/sriov/list_test.go:158]: Can you add nsName: "" ?

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**